include/alpaka/version.hpp

```diff
@@ -0,0 +1,26 @@
+/**
+* \file
+* Copyright 2016 Erik Zenker
+*
+* This file is part of alpaka.
+*
+* alpaka is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Lesser General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* alpaka is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU Lesser General Public License for more details.
+*
+* You should have received a copy of the GNU Lesser General Public License
+* along with alpaka.
+* If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define ALPAKA_VERSION_MAJOR 0
+#define ALPAKA_VERSION_MINOR 1
+#define ALPAKA_VERSION_PATCH 0
```

Comment:
Could you please add the lines you removed from alpaka.hpp? I find the `ALPAKA_VERSION` using the predef version macros really helpful. However, they should now use the major, minor and patch defines. You are missing the trailing newlin.